packages/Python/lldbsuite/test/lang/swift/variables/bridged_string/main.swift

```diff
@@ -13,12 +13,12 @@ import Foundation
 
 func main()
 {
-	var s1 = "Hello world"
-	var s2 = "ΞΕΛΛΘ"
-	var s3 = s1 as NSString
-	var s4 = s2 as NSString
-	var s5 = "abc" as NSString
-	var s6 = String(s5)
+	let s1 = "Hello world"
+	let s2 = "ΞΕΛΛΘ"
+	let s3 = s1 as NSString
+	let s4 = s2 as NSString
+	let s5 = "abc" as NSString
+	let s6 = String(s5)
```

Comment:
`s6` is the `is_Cocoa_or_small && is_a_value` case

Reply:
It's a little weird that the test case has 6 variables in it but it only tests s3-s6.  s1 & s6 are probably the same, but we only test the NSString version of the non-ascii String (s2)  Does that one work?

Reply:
Thanks for the catch! I didn't bother adding these checks because they are covered in another test, but it makes sense to have them for completeness. I've just pushed the change.

Reply:
Oh, yes, while you are messing with this, lines 45-67 of that test case can be replaced with:
 
    	        (target, process, thread, bkpt) = lldbutil.run_to_source_breakpoint(self,
    	                                   "Set breakpoint here", self.main_source_spec)
 
Be nice to do this if possible whenever we touch a test case.  This is much easier to read and would have meant hundreds fewer test case changes in Adrian's work to move the build out of tree.

Reply:
Done. And thanks, it's good to have this reminder!